samples/name_domains_values/Coded_Domain_Reliability.csv

```diff
@@ -1,7 +1,7 @@
 Name,Value
-Completely Reliable,A
-Usually Reliable,B
-Fairly Reliable,C
-Not Usually Reliable,D
-Unreliable,E
-Reliability Cannot Be Judged,F
+"Completely Reliable",A
```

Comment:
Please make the quotes apply to just that one .csv then - rather than changing everything at this point - for all the oher csv's we agreed **not** to use certain characters (including commas,semicolons,colons,quotes,etc. see the original OneNote spec from last year & see if we need to put that documentation in this repo for posterity)